cmd/kube-spawn/setup.go

```diff
@@ -106,6 +106,14 @@ func doSetup(numNodes int, baseImage string) {
 		}
 	}
 
+	// NOTE: workaround for making kubelet work with port-forward.
+	// Ideally we should solve the port-forward issue by either
+	// creating general add-ons based on torcx, or creating our own
+	// container image, or at least building socat statically on our own.
+	if err := bootstrap.DownloadSocatBin("./k8s"); err != nil {
```

Comment:
@iaguis Interesting, is kubelet's $PATH different from the default one? I'll change `/opt/bin/socat` to `/usr/bin/socat`.
I'll put the binary into `.kube-spawn/extras` as well.